riot-lorawan/downlink/README.md

```diff
@@ -0,0 +1,29 @@
+# Downlinks from the LoRaWAN network
+
+## **Goal**
+Write an application that prints a message when it receives a downlink
+ from the LoRaWAN network
+
+## Instructions
+  - Reuse ~/riot-course-exercises/riot-lorawan/lpp` to make LED1 blink
+    when receiving a message from the LoRaWAN network.
+  - Send downlink messages to your node from the TTN network: go to
+    `Devices> [your_device] > Downlinks`
+  - **Optional**: Use the TTN MQTT broker to send and receive data from
+    your node. Check the appendix below.
+
+## Appendix: Using the MQTT API
+  - The Mosquitto MQTT Client is used.
+  - Listening to uplink messages (device to network):
+```
+$ mosquitto_sub -h eu.thethings.network -p 1883 -u <username> -P <password>
```

Comment:
The mosquitto-clients should be updated with the versions that uses TLS authentication (download the .pem file and use port 8883). Otherwise that may not work all the time.

Reply:
where should this be notified? is it included in the VM? If so, is it only changing the port?

Reply:
The port needs to be changed and a new option --key with the path to the pem file added.

Reply:
maybe we can just point to this link? https://www.thethingsnetwork.org/docs/applications/mqtt/quick-start.html

Reply:
what do you think?

Reply:
yes, makes sense. And it's simpler